doc/userguide/rules/smtp-keywords.rst

```diff
@@ -34,7 +34,7 @@ smtp.command_line
 A single line from the client to the server. Multi-line commands will have a frame per
 line. Lines part of the SMTP DATA transfer are excluded.
 
-.. container:: example fule
+.. container:: example rule
```

Comment:
Seems to be missing a hyphen. example**-**rule as defined in `_static/css/suricata.css`